README.md

```diff
@@ -1,12 +1,33 @@
-# GGIRCS Industry Portal
 
-[![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=bcgov_cas-ciip-portal&metric=alert_status)](https://sonarcloud.io/dashboard?id=bcgov_cas-ciip-portal)
-[![CircleCI](https://circleci.com/gh/bcgov/cas-ciip-portal.svg?style=svg)](https://circleci.com/gh/bcgov/cas-ciip-portal)
-[![CII Best Practices](https://bestpractices.coreinfrastructure.org/projects/3459/badge)](https://bestpractices.coreinfrastructure.org/projects/3459)
+# Welcome to the CIIP Portal
 
-Readme WIP
+  
 
-## Backend
+CIIP stands for the **CleanBC Industrial Incentive Program** and is part of the CleanBC Program for Industry, which applies to large industrial operations that report their emissions under the Greenhouse Gas Industrial Reporting and Control Act (GGIRCA). The CIIP helps eligible industrial operations across the province by reducing carbon tax costs for facilities based on product-specific emissions benchmarks.
+
+See the [Documentation](#documentation) section below for more information and links.
+
+  
+  
+
+## Description
+
+CIIP requires an application form to be submitted annually. CIIP is a voluntary program created for industrial operations in B.C. Operators must submit a CIIP application form by the program application deadline (announced by the program area). As part of the application, information about an industrial facility's energy use, emissions, and production is required. This project is the portal to that application, which upon approval, leads to the **GGIRCS** (Greenhouse Gas Industrial Reporting and Control System) for the Climate Action Secretariat (CAS).
+
+
+
+## Status
+
+* Quality Gate: [![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=bcgov_cas-ciip-portal&metric=alert_status)](https://sonarcloud.io/dashboard?id=bcgov_cas-ciip-portal)
+
+* CircleCI: [![CircleCI](https://circleci.com/gh/bcgov/cas-ciip-portal.svg?style=svg)](https://circleci.com/gh/bcgov/cas-ciip-portal)
+
+* CII Best Practices: [![CII Best Practices](https://bestpractices.coreinfrastructure.org/projects/3459/badge)](https://bestpractices.coreinfrastructure.org/projects/3459)
+
```

Comment:
Just a thought: badges are meant to be quick indicators of quality control status for the dev-interested (who tend not to scroll far past the section of interest - the file tree) on scanning the repo homepage.

Most repos put them in a row at the top of the page, as prominent accountability badges of honor/shame 
eg: [React](https://github.com/facebook/react) [Vue](https://github.com/vuejs/vue) [Angular](https://github.com/angular/angular) 

It shouldn't diminish the readme title/intro if we just kept them in a neat row above the title.

Reply:
agreed